core/src/main/java/pl/agh/edu/engine/bank/TransactionType.java

```diff
@@ -1,10 +1,33 @@
 package pl.agh.edu.engine.bank;
 
 import pl.agh.edu.serialization.KryoConfig;
+import pl.agh.edu.utils.LanguageString;
 
 public enum TransactionType {
-	INCOME,
-	EXPENSE;
+	RECEIVED_CREDIT(true, "transaction.type.receivedCredit"),
+	ROOM_RENTING_INCOME(true, "transaction.type.roomRentingIncome"),
+	ROOM_DOWNGRADE_INCOME(true, "transaction.type.roomDowngradeIncome"),
+	ATTRACTION_DOWNGRADE_INCOME(true, "transaction.type.attractionDowngradeIncome"),
```

Comment:
tutaj też luźny pomysł żeby rozbić na poszczególne atrakcje. Tylko jeśli byłyby to osobne typy to potrzebne będą też kategorie transakcji